zerver/lib/actions.py

```diff
@@ -5008,6 +5022,9 @@ def do_create_realm(
             realm.demo_organization_scheduled_deletion_date = (
                 realm.date_created + datetime.timedelta(days=settings.DEMO_ORG_DEADLINE_DAYS)
             )
+
+        # Changing default permission policy for education organisations
```

Comment:
Once you rename the function as suggested above, this comment should no longer be needed.